deploy/stacks/container.py

```diff
@@ -181,10 +181,11 @@ def __init__(
 
     @run_if(['modules.s3_datasets.active', 'modules.dashboards.active'])
     def add_catalog_indexer_task(self):
```

Comment:
nit: container_id is only used in line 188, that change is not really needed

Reply:
Also used L213